src/locales/en_US.js

```diff
@@ -884,6 +884,11 @@ const strings = {
   confirm_continue_modal_body: 'Please confirm your understanding below:',
   confirm_continue_modal_button_text: 'I understand and agree to the terms',
 
+  // Gateway Warning Modal
+  gateway_agreement_modal_title: '%s Gatway Address Detected',
+  gateway_agreement_modal_body:
+    'This address is only valid for 24 hours and can only be used once.\n\nDo not send funds after the address has expired and/or send multiple transactions to this address.',
```

Comment:
Is 24 hours currency-specific? Also, we can just use 'or' instead of 'and/or' I have a suspicion that won't translate well

Reply:
"Is 24 hours currency-specific?" I don't think so.
I do recall seeing longer times but never shorter, so it's better be safe.

And I kinda like the 'and/or', very accurate ;)